### PR TITLE
PR Added Slot section to ComboBox and ChoiceBox

### LINKED ISSUE #225: Document ComboBox and ChoiceBox supported slots
https://github.com/webforj/webforj/pull/714

### PR DESCRIPTION
**Pull request by bbrennanbasis**
- Added a section to the ComboBox and ChoiceBox pages talking about the prefix and suffix slots
-This PR closes #225.

### CODE REVIEW

**Comment by bbrennanbasis** [comment]
@MatthewHawkins If the phrasing of these sections looks good to you, I'll make similar edits for Issues #224 and #222.

**Review by MatthewHawkins — DISMISSED** [review]

File: docs/components/lists/choice-box.md [hunk 1]
```diff
@@ -91,6 +91,17 @@ The `ChoiceBox` component has methods that allow manipulation of the dropdown di
...
 :::
 
+## Slots
+
+Slots provide flexible options for improving the functionality of a `ChoiceBox`. You can have icons, labels, loading spinners, clear/reset capability, avatar/profile pictures, and other beneficial components nested within a `ChoiceBox` to further clarify intended meaning to users.
+The `ChoiceBox` has two slots: the `prefix` and `suffix` slots. Use the `setPrefixComponent()` and `setSuffixComponent()` methods to insert various components before and after the displayed option within a `ChoiceBox`.
+
+```java
+ComboBox choiceBox = new ChoiceBox());
+  choiceBox.setPrefixComponent(new Img("http://localhost:8888/joystick.png"));
```
Comment (hyyan) [inline docs/components/lists/choice-box.md:101]: Please the usage of `Img` with Icons

**Review by hyyan — CHANGES_REQUESTED** [review]

**Comment by hyyan** [comment]
@bbrennanbasis please rebase and resolve conflicts

**Comment by bbrennanbasis** [comment]
@hyyan ok, it should be rebased now

**Review by hyyan — APPROVED** [review]

**Review by hyyan — APPROVED** [review]

**Pull request merged**